playbooks/roles/ecomworker/defaults/main.yml

```diff
@@ -138,6 +138,7 @@ ECOMMERCE_WORKER_SERVICE_CONFIG:
 
 
 ecommerce_worker_environment:
+  TZ: 'Australia/Melbourne'
```

Comment:
make this
`TZ: '{{ ECOMMERCE_WORKER_TZ }}'`

Reply:
Fixed.